dev-client/src/screens/ProjectSitesScreen.tsx

```diff
@@ -261,19 +270,28 @@ export function ProjectSitesScreen({
           options={sortingOptions}
         />
       </ListFilterModal>
-      <SiteCardList />
+      <SiteCardList showButtons={showButtons} />
     </ListFilterProvider>
   );
 
   return (
     <VStack m={3} pb={5} space={3} h="100%">
-      {isEmpty && <Text>{t('projects.sites.empty')}</Text>}
-      <Button
-        onPress={transferCallback}
-        alignSelf="flex-start"
-        _text={{textTransform: 'uppercase'}}>
-        {t('projects.sites.transfer') ?? ''}
-      </Button>
+      {isEmpty && (
+        <>
+          <Text>{t('projects.sites.empty')}</Text>
+          <RestrictByProjectRole role={['manager', 'contributor']}>
+            <Text>{t('projects.sites.empty_not_viewer')}</Text>
+          </RestrictByProjectRole>
+        </>
+      )}
+      <RestrictByProjectRole role={['manager', 'contributor']}>
+        <Button
+          onPress={transferCallback}
+          alignSelf="flex-start"
+          _text={{textTransform: 'uppercase'}}>
+          {t('projects.sites.transfer') ?? ''}
```

Comment:
suggestion: use the default value for `t()`.
```suggestion
          {t('projects.sites.transfer', '')}
```